### PR TITLE
Create job ci-kubernetes-e2e-kops-aws-sig-cli

### PR DESCRIPTION
**Pull request by jennybuckley**

### CODE REVIEW

File: jobs/env/ci-kubernetes-e2e-kops-aws-sig-cli.env [hunk 1]
```diff
@@ -0,0 +1 @@
+KOPS_PUBLISH_GREEN_PATH=gs://kops-ci/bin/latest-ci-green.txt
```
Comment (krzyzacy) [inline jobs/env/ci-kubernetes-e2e-kops-aws-sig-cli.env]: you don't want to overwrite the last-ci-green, I think just get rid of it

Comment (jennybuckley) [reply jobs/env/ci-kubernetes-e2e-kops-aws-sig-cli.env]: I just copied it directly from jobs/env/ci-kubernetes-e2e-kops-aws.env so the environment would be the same but I think you're right that I should just get rid of it

File: jobs/env/ci-kubernetes-e2e-kops-aws-sig-cli.env [hunk 2]
```diff
@@ -0,0 +1 @@
+KOPS_PUBLISH_GREEN_PATH=
```
Comment (krzyzacy) [inline jobs/env/ci-kubernetes-e2e-kops-aws-sig-cli.env]: just delete the env file :-)

**Pull request merged**